provider/baked/src/export.rs

```diff
@@ -355,14 +358,12 @@ impl DataExporter for BakedExporter {
         id: DataIdentifierBorrowed,
         payload: &DataPayload<ExportMarker>,
```

Comment:
Thought: Why doesn't `payload` get moved here? Then we wouldn't need the `Arc`. I guess it's because multi-export provider can call multiple exporters?

Reply:
Yes in general we don't need ownership